### PR TITLE
Add UITest for the Flow

### PR DESCRIPTION
**Pull request by 30mar**
## Description
UITests for The flow.
Unit tests are done with the original PR

## Ticket
https://jira-ibs.wbs.net.sap/browse/EXPOSUREAPP-8189

### CODE REVIEW

File: src/xcode/ENA/ENA/Source/View Helpers/AccessibilityIdentifiers.swift [hunk 1]
```diff
@@ -454,6 +461,9 @@ enum AccessibilityIdentifiers {
 	}
 	
 	enum Statistics {
+		enum LocalStatistics {
+			
+		}
```
Comment (whiskey) [inline src/xcode/ENA/ENA/Source/View Helpers/AccessibilityIdentifiers.swift:466]: Needed?

File: src/xcode/ENA/ENAUITests/Home/ENAUITests-Statistics.swift [hunk 2]
```diff
@@ -18,6 +18,30 @@ class ENAUITests_01b_Statistics: CWATestCase {
...
+		XCTAssertTrue(app.tables[AccessibilityIdentifiers.LocalStatistics.selectState].waitForExistence(timeout: .short))
+		// Tap on some data entry. Then we should be on select district screen.
+		app.cells.element(boundBy: 1).waitAndTap()
+		XCTAssertTrue(app.tables[AccessibilityIdentifiers.LocalStatistics.selectDistrict].waitForExistence(timeout: .short))
+		// Tap on some data entry. Then we should be back on the homescreen.
+		app.cells.element(boundBy: 14).waitAndTap()
+		// the Local statistics card will appear.
+		XCTAssertTrue(app.tables[AccessibilityIdentifiers.Home.tableView].waitForExistence(timeout: .medium))
+//		app.swipeDown(velocity: .slow)
+//		XCTAssertTrue(self.app.staticTexts[localStatisticsViewTitle].waitForExistence(timeout: .medium))
```
Comment (whiskey) [inline src/xcode/ENA/ENAUITests/Home/ENAUITests-Statistics.swift:42]: needed?

**Review by whiskey — CHANGES_REQUESTED** [review]

File: src/xcode/ENA/ENA/Source/View Helpers/AccessibilityIdentifiers.swift (hunk 1 above)
Comment (30mar) [reply src/xcode/ENA/ENA/Source/View Helpers/AccessibilityIdentifiers.swift:466]: no i moved it and forgot to delete this one, will push again

File: src/xcode/ENA/ENAUITests/Home/ENAUITests-Statistics.swift (hunk 2 above)
Comment (30mar) [reply src/xcode/ENA/ENAUITests/Home/ENAUITests-Statistics.swift:42]: This is the failing test that i asked about in the discussion Please double check why isn't it working if you have time

File: src/xcode/ENA/ENA/Source/Scenes/Home/Cells/Statistics/AddStatisticsCardView/AddStatisticsCardView.xib [hunk 3]
```diff
@@ -74,6 +74,7 @@
             <freeformSimulatedSizeMetrics key="simulatedDestinationMetrics"/>
             <connections>
                 <outlet property="addLocalIncidenceLabel" destination="wcn-ks-Lju" id="cbQ-XD-0jy"/>
+                <outlet property="addLocalIncidencesButton" destination="ypy-E2-Kp1" id="o63-Vp-zsm"/>
```
[the remark this answers is not in the document]
Comment (30mar) [reply src/xcode/ENA/ENA/Source/Scenes/Home/Cells/Statistics/AddStatisticsCardView/AddStatisticsCardView.xib:77]: It is just an IBOutlet so i can assign the Accessibility Identifiers. but yes probably will need to wire it again with the new UI

File: src/xcode/ENA/ENAUITests/Home/ENAUITests-Statistics.swift (hunk 2 above)
Comment (whiskey) [reply src/xcode/ENA/ENAUITests/Home/ENAUITests-Statistics.swift:42]: Ok. You can add a TODO for this next time and ignore the CI yelling at you :)

**Review by whiskey — APPROVED** [review]

**Pull request merged**